qa/tests/test_account_page.py

```diff
@@ -0,0 +1,45 @@
+from qa.PageObjects.AccountPage import AccountPage
+from qa.PageObjects.LoginPage import LoginPage
+from qa.utilities.BaseClass import BaseClass
+
+
+class TestAccountPage(BaseClass):
+
+    def test_update_username_and_email(self):
+        """Updating username and email"""
+        self._successful_log_in()
+        self.driver.get("http://127.0.0.1:5000/account")
```

Comment:
Move URLs to one place, e.g. env variables or config.py file with global used variables